modules/apps/static/portal-osgi-web/portal-osgi-web-wab-generator/build.gradle

```diff
@@ -4,8 +4,8 @@ targetCompatibility = "1.8"
 dependencies {
 	compileInclude group: "biz.aQute.bnd", name: "biz.aQute.bndlib", version: "3.3.0"
 
-	provided group: "com.liferay", name: "com.liferay.ant.bnd", version: "2.0.42"
```

Comment:
We have to exclude these 2, since they're embedded in the `portal-osgi-web-wab-generator` via the `-conditionalpackage` instruction in the `bnd.bnd` file.